docs/plugins/building_a_plugin/best_practices.md

```diff
@@ -8,42 +7,65 @@ affect the ability to install or use your plugin effectively.
 
 
 (best-practices-no-qt-backend)=
-
-## Don't include `PySide2` or `PyQt5` in your plugin's dependencies.
+## Don't include `napari[all]`, `PySide2`, `PyQt5` or `PyQt6` in your plugin's dependencies.
```

Comment:
Maybe `## Avoid including any form of Qt in your plugin's dependencies.

Reply:
I think I prefer the heading being explicit, but I added your line directly below it.